TuringLang/MCMCChains.jl
# Fix doctests (#342)

* Fix doctests

* Fix Github action

* Ensure that tests and docs use same package versions

* Rearrange test dependencies

* Use DecisionTree instead of XGBoost

* Only run doctests on 64bit architectures
